skeleton_chrome/content-script.js

```diff
@@ -26,10 +26,18 @@
   }
 
   // let ember-debug know that content script has executed
+  if (document) {
+    if (document.documentElement) {
+      if (!document.documentElement.dataset) {
+        document.documentElement.dataset = {};
+      }
+    } else {
+      document.documentElement = {};
+      document.documentElement.dataset = {};
+    }
+  }
   document.documentElement.dataset.emberExtension = 1;
```

Comment:
I think if there's not `documentElement` or dataset then it's certainly not an Ember app. We can skip setting `emberExtension = 1` altogether. I think it's better than mutating the document.